src/main/kotlin/no/skatteetaten/aurora/boober/service/resourceprovisioning/DatabaseSchemaProvisioner.kt

```diff
@@ -33,13 +35,25 @@ data class DbhUser(val username: String, val password: String, val type: String)
 
 data class DatabaseInstance(val port: Long, val host: String?)
 
+data class DbhError(val errorMessage: String) {
```

Comment:
Jeg lurer på om jeg ville unnlatt å eksponere denne dataklassen og heller bare laget en metode som henter feilmelding fra Dbhresponsen. Tatt i betraktning at DbhError nå faktisk representerer feilresponsen fra dbh, samt feilen som oppstår dersom feilresponsen ikke lar seg parse (double duty, uten at det er mulig å programmelt skille på hva som har oppstått; kommer Unknown fra dbh-responsen eller som en konsekvens av at dbh-responsen ikke lot seg parse?). DbhError kan da være en dataklasse med scope internt til metoden som parser responsen.

Kanskje også kalle den DbhErrorResponse for å være litt tydligere på hva den representerer?

Reply:
En metode som returerer feilmeldigen er en god løsning. Bør endre navne på klassen også.